src/scenes/Projects/Overview/ProjectOverview.tsx

```diff
@@ -218,6 +218,20 @@ export const ProjectOverview: FC = () => {
               </Grid>
             </Tooltip>
           )}
+          {projectOverviewData?.project.__typename === 'InternshipProject' && (
+            <Grid item>
+              <DisplaySimpleProperty
+                loading={!engagementListData}
+                label="Total Interns"
+                value={formatNumber(
+                  engagementListData?.project.engagements.total
```

Comment:
Is it possible for a project to have both Language and Internship Engagements? If so, this total might be incorrect

Reply:
Pretty sure a translation project only has language engagements and internship project only has internship engagements.  We've also enforced that creation on the client